README.md

```diff
@@ -93,6 +93,7 @@ This library currently supports the following cryptocurrencies and address forma
  - LUNA (bech32)
  - MONA (base58check P2PKH and P2SH, and bech32 segwit)
  - NANO (nano-base32)
+ - NEAR (base58, no check)
```

Comment:
plain text? or just remove this comment?

Reply:
Resolved. Thanks